load/LPTSExtractReader.py

```diff
@@ -684,6 +720,26 @@ def Reg_Recording_Status(self):
 	def Reg_StockNumber(self):
 		return self.record.get("Reg_StockNumber")
 
+	def StockNumberByFilesetIdAndType(self, filesetId, typeCode):
+		damIds = self.DamIdList(typeCode)
+		for (damId, _, _, fieldname) in damIds:
+			if filesetId == damId:
+				if LanguageRecord.ND_StockNumberDamIdIndex.get(fieldname) == True:
```

Comment:
note that there are some ND stocknumbers that have " none" or something similar. These should be treated as if the ND stocknumber is not even included.